From e8f469317a856910760b79dafcbb7c75970fb418 Mon Sep 17 00:00:00 2001
From: obarun <eric@obarun.org>
Date: Tue, 21 Apr 2020 13:26:02 +1100
Subject: [PATCH 2/4] remove oblist.h headers

---
 src/include/66/parser.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/include/66/parser.h b/src/include/66/parser.h
index b4f430c..1f966a7 100644
--- a/src/include/66/parser.h
+++ b/src/include/66/parser.h
@@ -20,10 +20,8 @@
 #include <sys/types.h>
 #include <stdint.h>
 
-#include <oblibs/oblist.h>
 #include <oblibs/mill.h>
 
-
 #include <skalibs/stralloc.h>
 #include <skalibs/genalloc.h>
 #include <skalibs/types.h>
-- 
2.14.1

