From fac486f1525305b65e62b59c956559c6e3301193 Mon Sep 17 00:00:00 2001
From: Vadym Kochan <vadim4j@gmail.com>
Date: Wed, 22 Apr 2020 06:26:35 +0300
Subject: [PATCH 3/4] tools/gen-deps: fix extra libs generation for exec files

Signed-off-by: Vadym Kochan <vadim4j@gmail.com>
---
 tools/gen-deps.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/gen-deps.sh b/tools/gen-deps.sh
index 6383ac2..27e5b3e 100755
--- a/tools/gen-deps.sh
+++ b/tools/gen-deps.sh
@@ -81,7 +81,7 @@ for dir in $(ls -1 src | grep -v ^include) ; do
       if echo $dep | grep -q -- \\.o$ ; then
         dep="src/$dir/$dep"
       fi
-      if echo $dep | grep -q -- '^\${.*_LIB}' ; then
+      if echo $dep | grep -q -e ^-l -e '^\${.*_LIB}' ; then
         libs="$libs $dep"
       else
         deps="$deps $dep"
-- 
2.14.1

